### PR TITLE
Update spells.xml

### PR DESCRIPTION
**Pull request by Lycanzito**
adding coldoown for these spells atleast;

### CODE REVIEW

**Comment by Lycanzito** [comment]
there is still missing the visual / action bar;

image

**Comment by gpedro** [comment]
Countdown - 40s (exaustion)
Group Countdown - 4s (group cd)

https://tibia.fandom.com/wiki/Eternal_Winter
https://tibia.fandom.com/wiki/Hell%27s_Core
https://tibia.fandom.com/wiki/Rage_of_the_Skies
https://tibia.fandom.com/wiki/Wrath_of_Nature

File: data/spells/spells.xml [hunk 1]
```diff
@@ -149,7 +149,7 @@
 		<vocation name="Druid" />
 		<vocation name="Elder Druid" />
 	</instant>
-	<instant group="attack" spellid="118" name="Eternal Winter" words="exevo gran mas frigo" lvl="60" mana="1050" prem="1" selftarget="1" exhaustion="40000" groupcooldown="4000" needlearn="0" script="attack/eternal winter.lua">
+	<instant group="special" spellid="118" name="Eternal Winter" words="exevo gran mas frigo" lvl="60" mana="1050" prem="1" selftarget="1" exhaustion="40000" groupcooldown="40000" needlearn="0" script="attack/eternal winter.lua">
```
Comment (gpedro) [inline data/spells/spells.xml:152]: 40s for both?

File: data/spells/spells.xml [hunk 2]
```diff
@@ -229,7 +229,7 @@
 		<vocation name="Druid" />
 		<vocation name="Elder Druid" />
 	</instant>
-	<instant group="attack" spellid="24" name="Hell's Core" words="exevo gran mas flam" lvl="60" mana="1100" prem="1" exhaustion="40000" groupcooldown="4000" selftarget="1" needlearn="0" script="attack/hells core.lua">
+	<instant group="special" spellid="24" name="Hell's Core" words="exevo gran mas flam" lvl="60" mana="1100" prem="1" exhaustion="40000" groupcooldown="40000" selftarget="1" needlearn="0" script="attack/hells core.lua">
```
Comment (gpedro) [inline data/spells/spells.xml:232]: same here

File: data/spells/spells.xml [hunk 3]
```diff
@@ -361,7 +361,7 @@
 		<vocation name="Knight" />
 		<vocation name="Elite Knight" />
 	</instant>
-	<instant group="attack" spellid="119" name="Rage of the Skies" words="exevo gran mas vis" lvl="55" mana="600" selftarget="1" prem="1" exhaustion="40000" groupcooldown="4000" needlearn="0" script="attack/rage of the skies.lua">
+	<instant group="special" spellid="119" name="Rage of the Skies" words="exevo gran mas vis" lvl="55" mana="600" selftarget="1" prem="1" exhaustion="400000" groupcooldown="4000" needlearn="0" script="attack/rage of the skies.lua">
```
Comment (gpedro) [inline data/spells/spells.xml:364]: 400000? LOL

File: data/spells/spells.xml [hunk 4]
```diff
@@ -477,7 +477,7 @@
 		<vocation name="Knight" />
 		<vocation name="Elite Knight" />
 	</instant>
-	<instant group="attack" spellid="56" name="Wrath of Nature" words="exevo gran mas tera" lvl="55" mana="700" prem="1" selftarget="1" exhaustion="40000" groupcooldown="4000" needlearn="0" script="attack/wrath of nature.lua">
+	<instant group="special" spellid="56" name="Wrath of Nature" words="exevo gran mas tera" lvl="55" mana="700" prem="1" selftarget="1" exhaustion="40000" groupcooldown="40000" needlearn="0" script="attack/wrath of nature.lua">
```
Comment (gpedro) [inline data/spells/spells.xml:480]: 40s for both?

**Review by gpedro — CHANGES_REQUESTED** [review]

**Comment by gerotib** [comment]
You want to achieve "unified" cooldown for both spells? I have it done for each spell like this in src, without any changes in spells.xml, idk if thats best solution tho. Code im using: 
Below this line:
https://github.com/opentibiabr/otservbr-global/blob/658d04b6ea203f669af048011d6868bd916cdde8/src/spells.cpp#L812
add:
```
Condition* seccond_condition = 0;
				switch (spellId)
				{
					//Hell's Core and Rage of the Skies
				case 119:
					seccond_condition = Condition::createCondition(CONDITIONID_DEFAULT, CONDITION_SPELLCOOLDOWN, cooldown, 0, false, 24);
					player->addCondition(seccond_condition);
					break;
				case 24:
					seccond_condition = Condition::createCondition(CONDITIONID_DEFAULT, CONDITION_SPELLCOOLDOWN, cooldown, 0, false, 119);
					player->addCondition(seccond_condition);
					break;
					//Swift Foot and Sharpshooter
				case 134:
					seccond_condition = Condition::createCondition(CONDITIONID_DEFAULT, CONDITION_SPELLCOOLDOWN, cooldown, 0, false, 135);
					player->addCondition(seccond_condition);
					break;
				case 135:
					seccond_condition = Condition::createCondition(CONDITIONID_DEFAULT, CONDITION_SPELLCOOLDOWN, cooldown, 0, false, 134);
					player->addCondition(seccond_condition);
					break;
					//Ultimate Terra Strike and Ultimate Ice Strike
				case 156:
					seccond_condition = Condition::createCondition(CONDITIONID_DEFAULT, CONDITION_SPELLCOOLDOWN, cooldown, 0, false, 157);
					player->addCondition(seccond_condition);
					break;
				case 157:
					seccond_condition = Condition::createCondition(CONDITIONID_DEFAULT, CONDITION_SPELLCOOLDOWN, cooldown, 0, false, 156);
					player->addCondition(seccond_condition);
					break;
					//Ultimate Flame Strike and Ultimate Energy Strike
				case 155:
					seccond_condition = Condition::createCondition(CONDITIONID_DEFAULT, CONDITION_SPELLCOOLDOWN, cooldown, 0, false, 154);
					player->addCondition(seccond_condition);
					break;
				case 154:
					seccond_condition = Condition::createCondition(CONDITIONID_DEFAULT, CONDITION_SPELLCOOLDOWN, cooldown, 0, false, 155);
					player->addCondition(seccond_condition);
					break;
					//Blood Rage and Protector
				case 132:
					seccond_condition = Condition::createCondition(CONDITIONID_DEFAULT, CONDITION_SPELLCOOLDOWN, cooldown, 0, false, 133);
					player->addCondition(seccond_condition);
					break;
				case 133:
					seccond_condition = Condition::createCondition(CONDITIONID_DEFAULT, CONDITION_SPELLCOOLDOWN, cooldown, 0, false, 132);
					player->addCondition(seccond_condition);
					break;
					//Wrath of Nature and Eternal Winter
				case 118:
					seccond_condition = Condition::createCondition(CONDITIONID_DEFAULT, CONDITION_SPELLCOOLDOWN, cooldown, 0, false, 56);
					player->addCondition(seccond_condition);
					break;
				case 56:
					seccond_condition = Condition::createCondition(CONDITIONID_DEFAULT, CONDITION_SPELLCOOLDOWN, cooldown, 0, false, 118);
					player->addCondition(seccond_condition);
					break;
				}

```

**Comment by gabz182** [comment]
Funciona, mas se vc troca para " Special" e usa uma magia o cooldown não aparece na action bar para outra magia que usa "Special" tbm.. assim como ta na imagem.

**Comment by dudantas** [comment]
Closing. This is not the correction, it is a gambiarra. The correct way is not to allow the player to have all the spells, only one of them at a time (and change them in the npcs)
But, thanks!

**Pull request closed without merging**